atom/renderer/api/atom_api_web_frame.h

```diff
@@ -37,7 +34,9 @@ class WebFrame : public mate::Wrappable<WebFrame> {
                              v8::Local<v8::FunctionTemplate> prototype);
 
  private:
-  explicit WebFrame(v8::Isolate* isolate);
+  explicit WebFrame(v8::Isolate* isolate,
```

Comment:
Its better to have two versions of `WebFrame` constructors rather than having an implicit definition for clarity.

```
WebFrame(v8::Isolate* isolate);
WebFrame(v8::Isolate* isolate, blink::WebFrame* web_frame);
```